File: dispatch_web/src/com/sunyard/dispatch/model/TimeChartModel.java
package com.sunyard.dispatch.model;

public class TimeChartModel {
	
	private Integer times;//最近第几次
	private Double timeHour;//执行时间
	private String startTime;//执行日期
	
	public String getStartTime() {
		return startTime;
	}
	public void setStartTime(String startTime) {
		this.startTime = startTime;
	}
	public Integer getTimes() {
		return times;
	}
	public void setTimes(Integer times) {
		this.times = times;
	}
	public Double getTimeHour() {
		return timeHour;
	}
	public void setTimeHour(Double timeHour) {
		this.timeHour = timeHour;
	}
}

File: dispatch_web/WebContent/app/model/dispatch/flow/TimeChartModel.js

Ext.define(projectName +'.model.dispatch.flow.TimeChartModel', {
    extend: 'Ext.data.Model',
    fields: [
		{name: 'times', type: 'int'},//第n次
		{name: 'timeHour',type: 'double'},//执行时间
		{name: 'startTime',type: 'string'}//开始执行时间
	]
	
});